tests/env.js

```diff
@@ -31,4 +37,19 @@ describe( 'Env', () => {
 			expect( isMac( 'foo' ) ).to.be.false;
 		} );
 	} );
+
+	describe( 'isEdge()', () => {
+		it( 'returns true for Edge UA strings', () => {
```

Comment:
Not a single real UA string ;)

Reply:
Yeah - I've copied from other test. I'm adding some real ones right now ;)

Reply:
OK... I was too slow :disappointed: